src/data/showcases.js

```diff
@@ -1094,6 +1094,15 @@ export const Showcases = [
     source: null,
     tags: [ "daotool"],
   },
+  {
+    title: "Great Empires",
+    description:
+      "Building ancient era metaverse on Cardano Blockchain. Phase 01 : Digital Collectible Card Game, Phase 02 : Metaverse, Utility token of the ecosystem GREMP",
+    preview: require("./showcase/great_empires.png"),
```

Comment:
Hi, It is done, Hope all okay now.

Reply:
got it... although if you could please shrink and/or more closely crop the image it will be a big help to keep our code base small... since it only displays in maximum 640p width anyway.  Yours is currently one of the largest images in the Showcase:
```
$ ls -l src/data/showcase | sort -nk5
...
-rw-rw-r-- 1 rphair rphair  849521 2022-09-20 21:58 great_empires.png
-rw-rw-r-- 1 rphair rphair  882488 2022-09-05 21:25 jesart.png
-rw-rw-r-- 1 rphair rphair  890369 2022-09-05 21:25 hypeskulls.png
-rw-rw-r-- 1 rphair rphair 1066777 2022-09-05 21:25 veritree.png
-rw-rw-r-- 1 rphair rphair 1244347 2022-09-20 19:36 book-token.png
-rw-rw-r-- 1 rphair rphair 1358213 2022-06-10 09:45 jetchicken.png
$
```
... but all of this detail won't render in the Dev Portal UI. 🧐

Reply:
Hi, I have reduced the pixel now at 640p and change the file name also. Could you please check now?

Reply:
@nithb thanks, especially for the tag updates.  We will be ready to go (pending other reviewers' approval of the addition itself) if:

1. you can remove the old, larger image (it's still there in your committed files)
2. if you rename the new, smaller image filename from `gremp.png` tp `great-empires.png`, which will also help the clarify of our repository & avoid any other reviews having problems with abbreviated filenames or underscore characters. 😎

Reply:
@rphair, I removed the old image and replaced the current image with great-empires name. 
I really appreciate your efforts ;-)..